### PR TITLE
Example install files for Elasticsearch and Python 3.

### PR DESCRIPTION
**Pull request by rsdoiel**
Anticipating that many of our projects may take advantage of modern Python or Elasticsearch I've included two example installation instructions. If those tools were requirements then they could be linked into README.md or INSTALL.md or if not needed removed.

### CODE REVIEW

**Review by mhucka — CHANGES_REQUESTED** [review]
See comments on files, below.

File: INSTALL-Elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+
+# Elasticsearch v7.x
+
+You can go to Elasticsearch's website and download the appropriate
+elastic search.
+
+    https://www.elastic.co/start
+
+Elasticsearch needs to be running locally to use Observatory. Observatory
```
Comment (mhucka) [inline INSTALL-Elasticsearch.md]: Observatory

File: INSTALL-Python3.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Otherwise you find yourself in the realm of confusing error messages.
+This documents focuses on getting Python 3 up and running on
+Debian/Ubuntu Linux based systems as well as on macOS via MacPorts.
+Your milleage may very.
+
+## Python versions issues
+
+It is possible to use your default OS provided python if it is version
+>= 3.6. Many distributions still ship with version 2.7 out of the box
+and pipy's pip corresponding to version 2.7. Observatory REQUIRES
```
Comment (mhucka) [inline INSTALL-Python3.md]: Observatory

**Review by mhucka — CHANGES_REQUESTED** [review]
This references Observatory, which is unfortunately not suitable for a general-purpose template. Could this text be changed to be more generic?

**Comment by rsdoiel** [comment]
Mike, I've updated the copy for both the Python and Elasticsearch. Let me know if you want additional changes. Thanks, Robert

**Pull request merged**